applications/EmpireApplication/tests/test_EmpireApplication.py

```diff
@@ -7,6 +7,7 @@
 
 from co_simulation_solver_test_factory import TestKratosSolver
 from co_simulation_solver_test_factory import TestSDoFSolver
+from co_simulation_solver_test_factory import TestSDoFStaticSolver
```

Comment:
You import the test but not use it, therefore the test will not be run. Please add the test. Check codacy

Reply:
done